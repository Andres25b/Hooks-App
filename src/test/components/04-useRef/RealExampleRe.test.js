import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';
import { RealExampleRe } from '../../../components/04-useRef/RealExampleRe';

describe('Pruebasn en <RealExampleRe />', () => {

    const wrapper = shallow( <RealExampleRe /> )

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
    });
    
    test('debe de mostrar el componente  <MultipleCustomHooks', () => {

        wrapper.find('button').simulate('click');

        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
    });
})
