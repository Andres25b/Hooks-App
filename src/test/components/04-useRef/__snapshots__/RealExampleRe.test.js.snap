// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebasn en <RealExampleRe /> debe mostrarse correctamente 1`] = `
<div
  className="container mt-4"
>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show / Hide
  </button>
</div>
`;
